gosu-test/src/test/gosu/gw/specContrib/statements/usesStatement/Errant_ExplicitStaticImportsAllowedByUnqualifiedClassNameFromTheSamePackage.gs

```diff
@@ -0,0 +1,15 @@
+package gw.specContrib.statements.usesStatement
```

Comment:
I did not express myself well.. so here is another try

If something on the parser side changes, and the static imports starts showing errors, we would not be able to catch the errors in tests unless the file name starts with Errant_

Please disregard the compilation point: The code will still compile because it is just the IDE error.

Here I am assuming that the highlighting errors / issuekeys (absence or presence) is caught only if the file name starts with Errant_ 
(Please correct me if I am wrong)
Thanks